sysutils/node_exporter/Makefile

```diff
@@ -1,6 +1,5 @@
 PLUGIN_NAME=		node_exporter
-PLUGIN_VERSION=		1.0
-PLUGIN_REVISION=	1
+PLUGIN_VERSION=		1.1
 PLUGIN_COMMENT=		Prometheus exporter for machine metrics
 PLUGIN_DEPENDS=		node_exporter
 PLUGIN_MAINTAINER=	dharrigan@gmail.com
```

Comment:
@jkellerer are you interested in becoming the maintainer? David is busy elsewhere it seems.

Reply:
@fichtner, yes sure. That’s fine for me.

Reply:
ok, just replace mail address in that maintainer line via commit

Reply:
done